client/components/Header/Header.js

```diff
@@ -217,51 +207,6 @@ class Header extends Component {
 							)}
 						</div>
 						<div className="buttons-wrapper">
-							{headerLinks.map((linkItem, index) => {
```

Comment:
Rather than adapting it to use the new `Menu`, I removed this code referencing `headerLinks` because this feature seems to be obsolete — there is no way to add them from the UI and no communities in prod currently make use of them.